README.md

```diff
@@ -2,7 +2,17 @@
 
 ## Purpose
 
-_... help fill this in!_
+* Define and maintain security policies and procedures for:
+  - the core Node.js project
+  - other projects maintained by the Node.js Foundation technical group
+* Define and maintain policies and procedures for the coordination of security concerns within the external Node.js open source ecosystem
```

Comment:
I'm not sure if this belongs in here, but I would at you may want to word this appropriately, perhaps has "offer to help" or something, as I wouldn't want this to cause friction within the module space. Sometimes there are different ways to fix a security issue, and different opinions on what is inherent to a API vs an actual security issue, so wouldn't want to see people feeling like the WG is imposing themselves & opinions down on modules like law outside of the foundation :)